assets/js/routes.js

```diff
@@ -215,20 +215,6 @@ function AppRouter ($stateProvider, $urlRouterProvider) {
       url: '/:accountIndex/transactions',
       views: transactionsViews
     })
-    .state('wallet.common.transaction', {
-      url: '/:accountIndex/transactions/:hash',
-      views: {
-        top: top,
-        left: {
-          templateUrl: 'partials/wallet-navigation.jade',
-          controller: 'WalletNavigationCtrl'
-        },
-        right: {
-          templateUrl: 'partials/transaction.jade',
-          controller: 'TransactionCtrl'
```

Comment:
Don't forget to remove `transaction.controller.js` and its test file if you're no longer using it.